scripts/ubuntu/build_openvslam_docker.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+TOPDIR=$PWD/../../ #openvslam directory
+
+( 
+    cd $TOPDIR
+
+    # Building Docker Image
+    docker build -t openvslam-desktop -f Dockerfile.desktop . --build-arg NUM_THREADS=`expr $(nproc) - 1`
+
+    # Starting Docker Container
+    # before launching the container, allow display access from local users
+    xhost +local:
+    # launch the container
+    docker run -it --rm --gpus all -e DISPLAY=$DISPLAY -v /tmp/.X11-unix/:/tmp/.X11-unix:ro --volume ${TOPDIR}/dataset:/dataset:ro --volume ${TOPDIR}/vocab:/vocab:ro openvslam-desktop
```

Comment:
Sorry. I overlooked this when I first looked at it, but it should be checked that nvidia-container-toolkit is installed to make it work in an environment without a GPU. Only if it exists, add `--gpus all`.

Reply:
I revised it.